app/Resources/translations/messages.en_GB.xliff

```diff
@@ -793,6 +830,12 @@ An e-mail with your activation code has been sent to the e-mail address %email%.
         <target>Number</target>
         <jms:reference-file line="44">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/SendSmsChallengeType.php</jms:reference-file>
       </trans-unit>
+      <trans-unit id="bec262808ffd307630f5d167bb7aaf470eabbe6b" resname="valid">
+        <source>valid</source>
+        <target>Match?</target>
```

Comment:
Maybe set this translation on the formfield label , and add a new translation that is set as the label for the checkox. Set that translation with something like:

`Is the value provided by %s correct?` Where %s would be the RV IdP name.

Also I would opt to translate `valid` to `Confirm`

Reply:
Personally I would like to have to form validated first before taking this effort. They probably need to be split up also and I think will not use labels but are translated in the AttributeMatchType template.

Reply:
In that case you might want to add some screenshots to the story and ask for input on the translations?

Reply:
I'll create an issue in Pivotal and assign it to Pieter. So it'll be non blocking for us right nowm and could address this later on.